sdk/appconfiguration/Azure.Data.AppConfiguration/src/ConfigurationClient.cs

```diff
@@ -869,92 +869,38 @@ private Request CreateGetRevisionsRequest(SettingSelector selector, string pageL
         /// Sets an existing <see cref="ConfigurationSetting"/> as read only in the configuration store.
         /// </summary>
         /// <param name="key">The primary identifier of the configuration setting.</param>
-        /// <param name="label">A label used to group this configuration setting with others.</param>
+        /// <param name="isReadOnly">true, to set an existing <see cref="ConfigurationSetting"/> as read only in the configuration store; otherwise, false.</param>
```

Comment:
If true, the `ConfigurationSetting` will be set to read only in the configuration store.  If false, it will be set to read write.

Reply:
Fixed